### PR TITLE
Adds drupal console

### PR DESCRIPTION
**Pull request by kimpepper**
...to the provisioning step.

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+curl -LSs http://drupalconsole.com/installer | php
+mv -v console.phar /usr/local/bin/drupal
```
Comment (nickschuch) [inline provision.sh]: I think well also need to make this executable?

Comment (kimpepper) [inline provision.sh]: Worked for me?

On 26 August 2015 at 11:51, Nick Schuch <notifications@github.com> wrote:

> In provision.sh
> <https://github.com/nickschuch/vd8/pull/34#discussion_r37939933>:
>
> > @@ -0,0 +1,4 @@
> > +#!/usr/bin/env bash
> > +
> > +curl -LSs http://drupalconsole.com/installer | php
> > +mv -v console.phar /usr/local/bin/drupal
>
> I think well also need to make this executable?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nickschuch/vd8/pull/34/files#r37939933>.
>



-- 
Kim Pepper
Technical Director
PreviousNext

desk: 02 8005 8670
mobile: 0414 212 530
skype: kim.pepper

Comment (nickschuch) [inline provision.sh]: Your not getting these?

```
[ ] Notice: The "phar.readonly" setting needs to be off to create Phars.
==> lamp: 
==> lamp:  [*] The "detect_unicode" setting is off.
==> lamp:  [*] The "allow_url_fopen" setting is on.
==> lamp:  [ ] Notice: The "apc.enable_cli" is on and may cause problems with Phars.
```

Comment (nickschuch) [inline provision.sh]: Doesn't look like they are too bad, just making sure we are getting the same.

Comment (kimpepper) [inline provision.sh]: Yeah. I got those.

Remove apc.enable_cli needs to be done in pnx-dev though, right?

**Comment by jibran** [comment]
In https://github.com/nickschuch/vd8/pull/41 we are updating the VM to https://atlas.hashicorp.com/pnx/boxes/lamp70 which already has this.

**Pull request closed without merging**